debian/couchdb.postrm

```diff
@@ -19,19 +19,30 @@ case $1 in
         if test -d "/opt/couchdb/etc"; then
             rm -rf "/opt/couchdb/etc"
```

Comment:
It's arguable whether we should keep the config file, because it includes the line that points at the data. I'm undecided on this point.

Reply:
If you're taking the mariadb example, you could grep out the data path from the ini files and use that to ensure we don't remove the data. But that's a lot of work for a simple shell script. I'm fine with leaving this line as is.